helper/keysutil/policy.go

```diff
@@ -739,7 +813,29 @@ func (p *Policy) VerifySignature(hashedInput []byte, sig string) (bool, error) {
 			Y:     keyParams.EC_Y,
 		}
 
-		return ecdsa.Verify(key, hashedInput, ecdsaSig.R, ecdsaSig.S), nil
+		return ecdsa.Verify(key, input, ecdsaSig.R, ecdsaSig.S), nil
+
+	case KeyType_ED25519:
+		sigBytes, err := base64.StdEncoding.DecodeString(splitVerSig[1])
```

Comment:
The base64 decoding could be moved outside of the switch since both cases need to decode